[Refactor] [Del 수정]

JeongYongKoo opened the pull request:
_fastPolicy 호출 부분이 주석처리 되어 있어 Enable 하였습니다.

JeongYongKoo committed:
[Refactor] [Del 수정]
_fastPolicy 호출 부분이 주석처리 되어 있어 Enable 하였습니다.

juyeon0park commented on ClarifyPJT/ClarifyPJT/Del.cpp:
```diff
@@ -82,6 +82,5 @@ NotFastDelete::deleteData(InputParameter input, DataManager* dataManager) const
 Del::Command(InputParameter input) {
-	// return _fastPolicy[static_cast<int>(input.column)]->deleteData(input, getDataManager());
-	return OutputParameter();
+	return _fastPolicy[static_cast<int>(input.column)]->deleteData(input, getDataManager());
```
확인했습니다!

elly-jung replied:
확인완료하였습니다!

DongyeonKwak reviewed (approved):
Delete enable 확인하였습니다.

JeongYongKoo committed:
[Fix] [Del 수정]
iter_swap 에서 rvalue 가 잘못 사용되어 build 안되는 문제 수정

[Test] [Del 수정]
-p 옵션일때 모든 entry 가 추가되도록 변경되어 TC 수정

simsonhyun commented on ClarifyPJT/TestProject/DeleteOption1Test.cpp:
```diff
@@ -77,7 +77,7 @@ TEST_F(DeleteOption1Test, DeleteAllByNameTestWithOptionP) {
 		EXPECT_EQ(aEmployee.GetName(), deleteEmployee.GetName());
 		deleteCount++;
 	}*/
-	EXPECT_EQ(5, result.resultCount);
```
TC 문제였군요 확인했습니다!
